### ISSUE TITLE
[Feature] Metrics on file/directory backup time

### ISSUE THREAD

**Issue by darvelo** [backuppc/backuppc#383]
I'm on Debian, and realizing now that BackupPC 3.3.2 has been the only supported version for years. I'm trying to determine whether to upgrade to 4.4.0.

For me, full backups for a couple of hosts are taking 3+ hours. I'd love to know if it's already possible, or if it can be developed, to gain insight on what files or directories take the longest time to backup, using some kind of metrics UI. The benefit would be knowing which files to move over to another host, in order to make it so I don't have to keep my laptop open for so long to complete a full backup.

Thank you for your continued work on BackupPC for so long. It's been so important to my life and work for years now.

**Comment by GWHAYWOOD** [comment]
I don't think that there's any immediate prospect of adding metrics to the UI, but you might be able to get what you need from the logs - perhaps just increasing the verbosity will do.  If it won't then it might be quick and easy to add a little extra log information.

What do you think?
